payloads/start.go

```diff
@@ -127,6 +127,9 @@ type RequestedResource struct {
 	// Value specifies the integer value associated with that resource.
 	Value int `yaml:"value"`
 
+	// ValueString is an optional string format value instead of integer
+	ValueString string `yaml:"value_string,omitempty"`
```

Comment:
@tpepper YAML does support Marshall and Unmarshall of custom types by implementing the interface https://github.com/go-yaml/yaml/blob/v2/yaml.go#L31

However how that interacts with versioning I am not sure. As the message may be versioned, but the type may not be versioned, unless we start implementing top level Marshaling support